cryptoki/tests/basic.rs

```diff
@@ -988,6 +1019,11 @@ fn test_clone_initialize() {
 #[test]
 #[serial]
 fn aes_key_attributes_test() -> TestResult {
+    /* FIXME: This is now broken in Kryoptic: https://github.com/latchset/kryoptic/issues/182 */
+    if !is_softhsm() {
```

Comment:
or maybe a `is_kryoptic` function, or a `test_pkcs11_implementation` returning an enum... don't want to be too annoying for the tests though 😅

Reply:
I  went with the `is_softhsm` as I do not expect the bugs in softhsm will get ever fixed anymore. I had some luck before 2021, but since then the project is mostly dead.

Reply:
> I do not expect the bugs in softhsm will get ever fixed anymore

ahh good to know!